src/test/java/com/glovoapp/planningpoker/e2e/logintosession/SessionIDEmptyTest.java

```diff
@@ -0,0 +1,32 @@
+package com.glovoapp.planningpoker.e2e.logintosession;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class SessionIDEmptyTest {
+
+    private final WebDriver driver = getDriver();
+    private MainPage mainPage = new MainPage(driver);
+    private String expecterResult = "Session ID must not be empty.";
```

Comment:
[IDE] Please pay more attention to [your IDE warnings](https://www.jetbrains.com/help/idea/code-inspection.html). This code shows:

> Field 'expecterResult' may be 'final' 

```suggestion
    private final String expectedResult = "Session ID must not be empty.";
```

Also, this constant should be `static`:

```suggestion
    private static final String EXPECTED_RESULT = "Session ID must not be empty.";
```

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)